interfaces/IBF-dashboard/src/app/components/chat/chat.component.ts

```diff
@@ -500,4 +502,76 @@ export class ChatComponent implements OnInit, OnDestroy {
       this.isWarn = false;
     }
   };
+
+  public isKenDrought() {
```

Comment:
const forecastMonthNumbers = this.country.countryDisasterSettings.find(
      (s) => s.disasterType === this.disasterType.disasterType,
    ).droughtForecastMonths;